### PR TITLE
Feature/map 111/add localization for backend

### PR DESCRIPTION
**Pull request by romawht22**
Added Ukrainian translation to all backend controllers using i18n.
What is i18n and how to use it: https://www.youtube.com/watch?v=P6HxWBox74g

### CODE REVIEW

**Review by obozhko-dev — COMMENTED** [review]
ок

**Review by obozhko-dev — APPROVED** [review]

File: frontend/src/components/Login/Login.tsx [hunk 1]
```diff
@@ -53,9 +53,9 @@ function Login() {
   const { errors } = useFormState({
     control
   });
-  const handleClickGoogle = () => {
-    window.open('http://localhost:3001/api/google', '_self');
-  };
+  // const handleClickGoogle = () => {
+  //   window.open('http://localhost:3001/api/google', '_self');
+  // };
```
Comment (Yojuks) [inline frontend/src/components/Login/Login.tsx]: Do we need those codes?

File: backend/index.ts [hunk 2]
```diff
@@ -11,9 +11,23 @@ import swaggerUI from 'swagger-ui-express';
...
+    fallbackLng: 'en',
+    backend: {
+      loadPath: './locales/{{lng}}/translation.json'
+    }
+  });
 
 const app = express();
 
+app.use(middleware.handle(i18next));
 // https://stackoverflow.com/questions/23259168/what-are-express-json-and-express-urlencoded
```
Comment (Yojuks) [inline backend/index.ts]: maybe you add some code why is this link there?

File: backend/controllers/LocationsController.ts [hunk 3]
```diff
@@ -44,23 +43,21 @@ const LocationsController = {
           locations.length < 50 ? locations.length : 50
         );
       }
+      if (!locations) {
+        return res.status(404).json({ message: req.t('locations_not_found') });
```
Comment (Yojuks) [inline backend/controllers/LocationsController.ts]: req.t  - I don't understand what this is - can we change t ->  translate?

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: backend/index.ts (hunk 2 above)
Comment (romawht22) [reply backend/index.ts]: this link was added by Yurii alongside with project skeleton

File: backend/controllers/LocationsController.ts (hunk 3 above)
Comment (romawht22) [reply backend/controllers/LocationsController.ts]: the "t" function itself comes form req(request object),so it must be this way

File: frontend/src/components/Login/Login.tsx (hunk 1 above)
Comment (romawht22) [reply frontend/src/components/Login/Login.tsx]: Deleted this function

**Review by Yojuks — APPROVED** [review]

File: backend/controllers/AuthController.ts [hunk 4]
```diff
@@ -83,21 +82,25 @@ const AuthController = {
       const isOk = await sendForgotPasswordMail(email, newPassword);
       if (!isOk) {
         return res.status(400).json({
-          error: 'An error occurred while sending the email...',
+          message: req.t('password_send_error'),
           success: false
```
Comment (6mahdapihka9) [inline backend/controllers/AuthController.ts]: The "message" field should be sent only on success
change it to "error"

**Review by 6mahdapihka9 — APPROVED** [review]

**Pull request merged**